DicesGame/GameRule.cs

```diff
@@ -8,34 +8,50 @@ namespace DicesGame
 {
     internal class GameRule
     {
+        public GameRule(string gamerule) { 
+            CurrentRule= gamerule;
+        }
         public string CurrentRule { get; private set; }
 
         public void ChangeGameRule()
         {
-            if (CurrentRule == "Game Rule: Lesser Than \"You must roll a total lower than the House's roll to win.\"")
+            if (CurrentRule == "Game Rule: You must roll a total lower than the House's roll to win.")
             {
-                CurrentRule = "Game Rule: Greater Than \"You must roll a total higher than the House's roll to win.\"";
+                CurrentRule = "Game Rule: You must roll a total higher than the House's roll to win.";
             }
             else
             {
-                CurrentRule = "Game Rule: Lesser Than \"You must roll a total lower than the House's roll to win.\"";
+                CurrentRule = "Game Rule: You must roll a total lower than the House's roll to win.";
```

Comment:
create a variable in the class that stores the option higher/lower
and also transform this into an one line code variable = condition ? result1 : result2;